modules/core/shared/src/main/scala/lucuma/core/geom/GmosScienceAreaGeometry.scala

```diff
@@ -16,6 +16,13 @@ import lucuma.core.math.syntax.int._
   */
 object GmosScienceAreaGeometry {
 
+  // base target
+  def base =
+    ShapeExpression.point(Offset.Zero)
+
+  def baseAt(posAngle: Angle, offsetPos: Offset) =
```

Comment:
By definition the base is always at `(0, 0)` so maybe `def pointAt` would be an appropriate name?